app/controllers/items_controller.rb

```diff
@@ -8,6 +8,13 @@ def create
     redirect_to list_path(@list)
   end
 
+  def update
+    @item = Item.find(params[:id])
+    toggle_state(@item)
+
+    redirect_to list_path(@item.list_id) # TODO: is there a shorthand?
```

Comment:
Is there an all-in-one path that I should use here?

Reply:
Not that I can think of, I may have seen the whole model passed to the path generator before but using the id is the most common usage and all good!